dist2src/cli.py

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/python3
 
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: MIT
```

Comment:
I remember some discussion about those shortcuts, that we can't use them. Maybe @TomasTomecek can remember?

Reply:
According to [this training](https://trainingportal.linuxfoundation.org/learn/course/licensing-basics-for-developers-lfc191/file-notices/file-notices?page=7), this is a completely valid way to reference the license: it's unambiguous and explicit.